src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -36,75 +22,88 @@ const ApplicationModal = ({
   const handleCancel = () => {
     setDisplayModal(false);
     setProfileId('');
-    setNotesValue(notes);
-    setHideForm(true);
   };
 
-  const displayForm = () => {
-    setHideForm(false);
-  };
+  const openNotificationWithIcon = (type, status, err) => {
+    if (type === 'success') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User has been approved successfully',
+        });
+      } else {
+        notification[type]({
+          message: 'User has been rejected successfully',
+        });
+      }
+    }
 
-  const handleChange = e => {
-    setNotesValue({
-      ...notesValue,
-      [e.target.name]: e.target.value,
-    });
+    if (type === 'error') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User could not be approved at this time',
+          description: `Error: ${err}`,
+        });
+      } else {
+        notification[type]({
+          message: 'User could not be rejected at this time',
+          description: `Error: ${err}`,
+        });
+      }
+    }
   };
 
-  const addNote = e => {
-    axiosWithAuth()
-      .put(
-        `/application/update-notes/${currentApplication.application_id}`,
-        notesValue
-      )
-      .then(res => {
-        updateModal();
-      })
-      .catch(err => {
-        console.log(err);
-      });
-    e.preventDefault();
-    setHideForm(true);
+  const approveRejectHelper = status => {
+    if (currentApplication.role_name === 'mentor') {
```

Comment:
currentApplication.accepting_new_mentees === undefined 
There is no role_name in mentor data

Reply:
We generate `role_name` on the FE when the table renders

Reply:
my bad @Johnjx , just saw the hasOwnProperty function, everything looks clean, great and coming to life with the status notifications, great work John